internal/types/types.go

```diff
@@ -45,9 +47,14 @@ var (
 
 	hasContextStyle = lipgloss.NewStyle().
 			Foreground(lipgloss.Color(hasContextColor))
+
+	TaskPrefixEmptyErr      = errors.New("Task prefix cannot be empty")
+	TaskSummaryBodyEmptyErr = errors.New("Task summary body is empty")
+	TaskSummaryTooLongErr   = fmt.Errorf("Task summary is too long; max length allowed: %d", TaskSummaryMaxLen)
```

Comment:
The canonical way to define error in Go is to use Err prefix


```suggestion
	ErrTaskPrefixEmpty      = errors.New("Task prefix cannot be empty")
	ErrTaskSummaryBodyEmpty = errors.New("Task summary body is empty")
	ErrTaskSummaryTooLong   = fmt.Errorf("Task summary is too long; max length allowed: %d", TaskSummaryMaxLen)
```